src/net/sf/memoranda/ui/App.java

```diff
@@ -136,10 +192,18 @@ void init() {
 		frame.setVisible(true);
 		frame.toFront();
 		frame.requestFocus();
-
 	}
 
 	public static void closeWindow() {
```

Comment:
Make sure that US-40's version of this method overwrites this version once merged.